docs/en/operations/storing-data.md

```diff
@@ -113,3 +113,44 @@ Example of disk configuration:
     </storage_configuration>
 </yandex>
 ```
+
+## Configuring WebServer {#configuring-webserver}
+
+Uses to store data on web server as static files (for example, table's data directory) using a disk with type `web` and to run queries on this data. It can be useful for serving public datasets.
```

Comment:
```suggestion
You can store data on a web server as static files (for example, table's data directory) using a disk with type `web` and run queries on this data. It can be useful for serving public datasets.
```